src/Migrations/Version20230502065100.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace OutputDataConfigToolkitBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230502065100 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Migrate columns with o_ prefix';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $table = $schema->getTable('bundle_outputdataconfigtoolkit_outputdefinition');
+
+        if ($table->hasColumn('o_id')) {
```

Comment:
Should we adapt existing [migration ](https://github.com/pimcore/output-data-config-toolkit/blob/5.x/src/Migrations/Version20221130082116.php#L37) instead of adding a new one as it already replaces `o_classId` to `classId`?

Reply:
If you want I can change it. But what if someone has already executed the migration?

Reply:
the migration was introduced in this PR https://github.com/pimcore/output-data-config-toolkit/pull/70 for v5 and never got into release.

Reply:
ok, I will change the migration